src/types/bridge-wallet.type.ts

```diff
@@ -0,0 +1,6 @@
+import { BIP32Interface } from 'bitcoinjs-lib';
+
+interface BridgeWallet extends BIP32Interface {
+  address: string;
+  passphrase: string;
+}
```

Comment:
This looks vaguely incorrect (we may not always know the `passphrase`), but it's also not used anywhere afaict. Probably don't want to add dead code.